iroh-relay/src/client.rs

```diff
@@ -211,94 +185,6 @@ impl ClientBuilder {
         Ok(conn)
     }
 
-    async fn connect_relay(
```

Comment:
`connect_relay` and all of its call graph dependencies are now moved into a submodule `iroh_relay::client::connect_relay`, this makes cfg-ing out much easier.